rmarkdowns/Week_3.Rmd

```diff
@@ -168,7 +228,19 @@ ggplot() +
   geom_sf(data = schools$osm_polygons) +
   geom_sf_text(data = schools$osm_polygons, aes(label = name), size = 2, hjust = 0) +
   coord_sf(xlim = c(-156.69726, -155.97909), ylim = c(20.57443, 21.03156), expand = FALSE)
+
 ```
+
+```{r}
+ggplot() +
+  geom_sf(data = healthcare$osm_polygons) +
+  geom_sf_text(data = healthcare$osm_polygons, aes(label = name), size = 2, hjust = 0) + 
+  coord_sf(xlim = c(-158.28058, -157.64863), 
+           ylim = c(21.25482,  21.71201),
+           expand = FALSE)
+```
```

Comment:
This looks right! Nice job so far. It will look better once we add more layers :)